install_config/redeploying_certificates.adoc

```diff
@@ -249,41 +252,54 @@ cluster configuration:
 - `openshift_master_cluster_hostname`
 - `openshift_master_cluster_public_hostname`
 
-[[redeploying-master-etcd-node-certificates-current-ca]]
-=== Redeploying Using the Current CA
+[[redeploying-all-certificates-current-ca]]
+=== Redeploying All Certificates Using the Current CA
+
+The *_redeploy-certificates.yml_* playbook does _not_ regenerate the
+{product-title} CA certificate. New master, etcd, node, registry, and router
+certificates are created using the current CA certificate to sign new
+certificates.
 
-By default, the *_redeploy-certificates.yml_* playbook does _not_ regenerate the
-{product-title} CA. New certificates are created using the current
-{product-title} CA.
+This also includes serial restarts of:
+
+- etcd
+- master services
+- node services
 
 To redeploy master, etcd, and node certificates using the current
-{product-title} CA, run the playbook, specifying your inventory file:
+{product-title} CA, run this playbook, specifying your inventory file:
 
 ----
 $ ansible-playbook -i <inventory_file> \
     /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-cluster/redeploy-certificates.yml
 ----
 
-[[redeploying-master-etcd-node-certificates-new-custom-ca]]
-=== Redeploying Using a New or Custom CA
+[[redeploying-all-certificates-new-custom-ca]]
+=== Redeploying All Certificates Using a New or Custom CA
 
-Setting `openshift_certificates_redeploy_ca=true` when running the
-*_redeploy-certificates.yml_* playbook causes the current {product-title} CA to
-also be regenerated and redeployed, and new certificates are created using the
-new CA.
+The *_redeploy-openshift-ca.yml_* playbook redeploys the {product-title} CA
+certificate by generating a new CA certificate and distributing an updated
+bundle to all components including client *_kubeconfig_* files and the node CA
+trust.
```

Comment:
I'm not sure of the "trust" reference. Is it supposed to be something like:

> The *_redeploy-openshift-ca.yml_* playbook redeploys the {product-title} CA certificate by generating a new CA certificate and distributing an updated bundle to all components including client *_kubeconfig_* files and the trusted node CA.

Unless I'm misunderstanding what "trust" is?